clients/venice-client/src/test/java/com/linkedin/venice/fastclient/meta/RequestBasedMetadataTestUtils.java

```diff
@@ -69,12 +69,15 @@ public static D2TransportClient getMockD2TransportClient(
       Schema storeValueSchema) {
     D2TransportClient d2TransportClient = mock(D2TransportClient.class);
 
+    Map<String, String> partitionerParams = new HashMap<>();
+    partitionerParams.put("testKey", "testValue");
```

Comment:
I understand this is a very simple change, but the test added does not test the problem this is trying to fix

Reply:
`RequestBasedMetadataTest#testMetadata` will fail without the fix. Discussed offline with Nisarg and resolving this.